src/servers/launcher/main.cpp

```diff
@@ -0,0 +1,849 @@
+#include <libpq-fe.h>
+
+#include <array>
+#include <cerrno>
+#include <chrono>
+#include <cstdint>
+#include <cstring>
+#include <expected>
+#include <iomanip>
+#include <iostream>
+#include <netinet/in.h>
+#include <ranges>
+#include <signal.h>
+#include <stdexcept>
+#include <sstream>
+#include <string>
+#include <sys/epoll.h>
+#include <sys/wait.h>
+#include <thread>
+#include <unistd.h>
+#include <utility>
+#include <variant>
+
+using namespace std::literals;
+using notManyMilliseconds = std::chrono::duration<int, std::milli>;
```

Comment:
I’m not sure this is a really useful alias. The standard `std::chrono::milliseconds` will probably be 64 bits, while using an `int` will probably be 32 bits, but… you don’t really save anything in practice, because the values will pretty much always be stored and passed in 64-bit registers. (Unless you’re planning to run your game server on a 32-bit microcontroller?)